config/application.rb

```diff
@@ -15,6 +15,8 @@
 
 module DcafCaseManagement
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.1
```

Comment:
Yeah it was a new value from rails app:update. I think it just tells it to load new_framework_defaults and then the version.